web/utils.py

```
@@ -414,6 +416,306 @@ def create_leaderboard_context(
+            customer = stripe.Customer.create(
+                email=user.email,
+                name=f"{user.first_name} {user.last_name}".strip(),
+                metadata={"user_id": user.id},
+            )
+
+            # Save the customer ID to the user membership
+            if hasattr(user, "membership"):
+                user.membership.stripe_customer_id = customer.id
+                user.membership.save(update_fields=["stripe_customer_id"])
+            else:
+                raise ValueError(f"User {user.email} does not have a membership object")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)